modules/RTC/TraceablePeerConnection.js

```diff
@@ -2108,6 +2108,67 @@ TraceablePeerConnection.prototype._adjustLocalMediaDirection = function(
     return localDescription;
 };
 
+/**
+ * Munges the stereo flag as well as the opusMaxAverageBitrate in the SDP, based
+ * on values set through config.js, if present.
+ *
+ * @param {RTCSessionDescription} description that needs to be munged.
+ * @returns {RTCSessionDescription} the munged description.
+ */
+TraceablePeerConnection.prototype._mungeOpus = function(description) {
```

Comment:
Maybe first check if stereo or opusMaxAverageBitrate were configured, to avoid the processing otherwise.